Stellar-transaction.x

```diff
@@ -514,27 +509,27 @@ case CONTRACT_ID_FROM_ASSET:
     Asset asset;
 };
 
-struct CreateContractArgs
+struct InstantiateContractArgs
 {
     ContractID contractID;
-    SCContractCode source;
+    SCContractExecutable executable;
 };
 
-union HostFunction switch (HostFunctionType type)
+union SmartContractOperation switch (SmartContractOperationType type)
 {
-case HOST_FUNCTION_TYPE_INVOKE_CONTRACT:
-    SCVec invokeArgs;
-case HOST_FUNCTION_TYPE_CREATE_CONTRACT:
-    CreateContractArgs createContractArgs;
-case HOST_FUNCTION_TYPE_INSTALL_CONTRACT_CODE:
-    InstallContractCodeArgs installContractCodeArgs;
+case SMART_CONTRACT_OPERATION_CALL_CONTRACT:
+    SCVec callArgs;
+case SMART_CONTRACT_OPERATION_INSTANTIATE_CONTRACT:
+    InstantiateContractArgs instantiateContractArgs;
```

Comment:
I like upload and instantiate over install and create. It's much easier to infer the order (create sounds like it should be called before install, which isn't true).